lib/service/version.rb

```diff
@@ -1,3 +1,3 @@
 module Service
-  VERSION = '3.30.0'
+  VERSION = '3.31.0'
```

Comment:
We've got a version collision between our two PRs, so let's either combine them or sequence them.